cookies/views/resource.py

```diff
@@ -63,9 +63,11 @@ def resource(request, obj_id):
         'content_relations': resource.content.filter(is_deleted=False),
         'part_relations': part_relations,
         'content_region_relations': content_region_relations,
+        'show_reupload': False
     }
 
     resource_pending_uploads = resource.giles_uploads.filter(state=GilesUpload.PENDING)
+    if resource.giles_uploads.count() - resource_pending_uploads.count() == 0: context['show_reupload'] = True
```

Comment:
do you mean:
```
resource.giles_uploads.count() == resource_pending_uploads.count()
```

Reply:
just turn this into a ternary operator and set 'show_uploads' to false otherwise (remove line 66). it's more elegant than setting it to false and set overriding it.